### PR TITLE
Added a Technology Preview module

### PR DESCRIPTION
**Pull request by ahardin-rh**

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@huffmanca PTAL :rocket:

File: modules/technology-preview.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+[IMPORTANT]
+====
+{FeatureName} is a Technology Preview release only. Technology Preview releases
+are not supported with Red Hat production service-level agreements (SLAs) and
+might not be functionally complete, and Red Hat does not recommend using them
+for production. These features provide early access to upcoming product
+features, enabling customers to test functionality and provide feedback during
+the development process. For more information see
+link:https://access.redhat.com/support/offerings/techpreview/ [Red Hat
```
Comment (huffmanca) [inline modules/technology-preview.adoc]: Need to remove the space here for the link to function correctly.

**Review by huffmanca — COMMENTED** [review]
One small change. Other than that it looks good to me!

File: modules/technology-preview.adoc (hunk 1 above)
Comment (ahardin-rh) [reply modules/technology-preview.adoc]: Whoops! Fixed. Thanks!

**Pull request merged**